jekyll/_cci2/skip-build.md

```diff
@@ -60,17 +68,20 @@ Date:   Tue Apr 25 15:56:42 2016 -0800
 
 When pushed to a VCS, this commit will not be built on CircleCI because of the `[ci skip]` or `[skip ci]` in the commit description.
 
-**Note:**
-If you push multiple commits at once, a single `[ci skip]` or `[skip ci]` will skip the build **for all commits**.
+## Auto cancelling
+{: #auto-cancelling}
 
-## Auto cancelling a redundant build
-{: #auto-cancelling-a-redundant-build }
-
-If you are frequently pushing changes to a branch, you increase the chances of queueing. This means you might have to wait for an older pipeline to finish building before the most recent version starts.
+If you are frequently pushing changes to a branch, you increase the chances of queueing. This means you might have to wait for an older pipeline to complete before the most recent version starts.
 
 To save time, you can configure CircleCI to automatically cancel any queued or running pipelines when a newer pipeline is triggered on that same branch.
 
-**Note:** Your project's default branch (usually `master`) will never auto-cancel builds.
+### Scope
+{: #scope }
+
+A few points to note regarding the use of the auto-cancel feature:
+
+* When using the auto-cancel feature, pipelines will still be created, but the included workflows (and therefore jobs) will not run.
```

Comment:
This sentence does not make sense to me. The workflows in the most recent will run, but *previous* workflows are cancelled.

Reply:
Yeah sorry that does not make sense! I'll just remove the line.